homeassistant/components/plugwise/climate.py

```diff
@@ -78,13 +85,33 @@ def current_temperature(self) -> float:
         return self.device["sensors"]["temperature"]
 
     @property
-    def target_temperature(self) -> float:
-        """Return the temperature we try to reach."""
-        return self.device["thermostat"]["setpoint"]
+    def target_temperature(self) -> float | None:
+        """Return the temperature we try to reach.
+
+        Connected to the HVACMode combination of AUTO-HEAT.
+        """
+
+        return self.device["thermostat"].get("setpoint")
```

Comment:
Why did this change to get? Isn't there always a set point?

Reply:
Because "setpoint" is not in the output when "setpoint_high" and "setpoint_low" are there. And vice versa.

Reply:
Right, so when does this happen? When is this method called in such cases?